src/components/Common/Loading.tsx

```diff
@@ -1,16 +1,23 @@
 const img = "/images/care_logo_gray.svg";
 
 const Loading = () => {
+  const isPatientsPage = window.location.pathname === "/patients";
+  const isAssetsPage = window.location.pathname === "/assets";
+  const isUserPage = window.location.pathname === "/users";
+
+  const minHeight =
+    isPatientsPage || isAssetsPage || isUserPage ? "500px" : "800px";
```

Comment:
hardcoding things inside a generic component based on where it's going to be used is not maintainable. 

Assume generic components to be built like a black-box. It itself shouldn't take decisions for itself based on where it'll be used, however you may pass props from the parent component for this component to behave in a particular desired way.